### ISSUE TITLE
Internal server error when resetting password

### ISSUE THREAD

**Issue by aptiko** [openmeteo/aira#27]
Go to lost password, type email address, click on the password reset link in the email, specify a new password, and click the button. Result:
```
NoReverseMatch at /password/reset/complete/
Reverse for 'aira.views.home' with arguments '()' and keyword arguments '{}' not found. 0 pattern(s) tried: []
```

**Comment by kickapoo** [comment]
It was only a forgotten `url` path. I deleted it as there is no need. User that `lost password` need only to `login`, not to navigate back to `home` page.

**Comment by aptiko** [comment]
When fixing bugs, please write `(fixes #27)` in the commit message instead of just `(#27)`, otherwise github does not close them.
